abac/src/main/java/no/nav/common/abac/AbacService.java

```diff
@@ -0,0 +1,138 @@
+package no.nav.common.abac;
+
+import no.nav.common.abac.domain.AbacPersonId;
+import no.nav.common.abac.domain.request.ActionId;
+import no.nav.common.abac.domain.request.XacmlRequest;
+import no.nav.common.abac.domain.response.Decision;
+import no.nav.common.abac.domain.response.XacmlResponse;
+import no.nav.common.abac.exception.PepException;
+
+import static no.nav.common.abac.XacmlRequestBuilder.*;
+
+public class AbacService implements Pep {
+
+    public final static String VEILARB_DOMAIN = "veilarb";
```

Comment:
Er dette en fil vi egentlig ønsker å videreføre? Får ikke akkurat en common-vibe av at første linje refererer til `veilarb` :P

Reply:
Snakket om akkurat dette med Matias. Er enig i at det teknisk sett ikke hører hjemme i common, men er usikker på om vi gidder å lage overbygg når det uansett kun er vi som bruker det. Det er enkelt nok å lage sin egen tjeneste som bruker AbacClient, så tenker at vi heller kan lage noe ekstra hvis behovet dukker opp.

Reply:
Men vi kunne vurdert å gjøre Pep om til VeilarbPep eller noe lignende, slik at det er tydligere at denne gjelder for veilarb domenet.

Reply:
Btw, det er ikke bare domenet, men et par andre ting i abac-modulen som f.eks attributter som også kun gjelder for oss

Reply:
Mye i abac-modulen som ikke burde vært der imho. :P Men støtter å evt rename  ting til `VeilArb******` for å signalisere at det er spesifikt for PTO.